code/modules/mob/living/carbon/human/human.dm

```diff
@@ -9,6 +9,7 @@
 	var/embedded_flag	  //To check if we've need to roll for damage on movement while an item is imbedded in us.
 	var/obj/item/weapon/rig/wearing_rig // This is very not good, but it's much much better than calling get_rig() every update_canmove() call.
 	mob_size = 9//Based on average weight of a human
+	var/malf_control = 0 // for IPC's only

```

Comment:
Can you make a static list in APC class for this?

```
var/static/list/hacked_ipcs = list()
```

And remove/add mobs in there whenever needed. And then in the places you want to check, you just run, `if (someIPC in hacked_ipcs)`.